Resources/doc/index.rst

```diff
@@ -4,9 +4,20 @@ FOSJsRoutingBundle
 This bundle allows to expose Symfony Routes to JavaScript, so you can generate
 relative or absolute URLs in the browser using the same routes as in the backend.
 
+.. _Installation: https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/blob/master/Resources/doc/installation.rst
+.. _Usage: https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/blob/master/Resources/doc/usage.rst
+.. _Commands: https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/blob/master/Resources/doc/commands.rst
+
+1. Installation_ 
+2. Usage_
+3. Commands_
```

Comment:
This will look weird. I suggest to use reStructuredText directives for this:

```rst
#. :doc:`Installation </installation.rst>`
#. :doc:`Usage </usage>`
#. :doc:`Commands </commands>`
```